api/requests/views.py

```diff
@@ -145,7 +144,7 @@ class PreprintRequestActionList(JSONAPIBaseView, generics.ListAPIView, PreprintR
     permission_classes = (
         permissions.IsAuthenticatedOrReadOnly,
         base_permissions.TokenHasScope,
-        MustBeModerator,
+        PreprintRequestPermission,
```

Comment:
Not super familiar, but the `PreprintRequestPermission` is broader, just checking that you're okay with opening this up to moderators, admins, and then it looks like preprint requester, so preprint's creator?

Reply:
Yeah, I saw that. I'm pretty sure the creator, for `PreprintRequestAction`s, is the the user who submitted the request. The target is set here:
https://github.com/CenterForOpenScience/osf.io/blob/6ab6f8d883b10f6f7949f62bee73756ad4b848a5/api/requests/permissions.py#L68-L69
and the target for a `PreprintRequestAction` is a `PreprintRequest`:
https://github.com/CenterForOpenScience/osf.io/blob/6ab6f8d883b10f6f7949f62bee73756ad4b848a5/osf/models/action.py#L53-L54
The requester is then pulled from the `PreprintRequest` creator:
https://github.com/CenterForOpenScience/osf.io/blob/6ab6f8d883b10f6f7949f62bee73756ad4b848a5/api/requests/permissions.py#L83
So this would make request actions available to the requester, which is what we need/want here.